### DISCUSSION TITLE
[Feature Request] Deactivate Alias from Email header without Login

### DISCUSSION THREAD

**Discussion by ayancey** [anonaddy/anonaddy#373] [Feature Requests]
![image](https://user-images.githubusercontent.com/10055792/197888534-93b8b44d-4e45-41d2-b1ba-c8f4423630ef.png)

It would be amazing if the deactivate alias endpoint (`https://app.anonaddy.com/deactivate/<uid>?signature=<sig>`) could work even when I am not logged in. At work and on my phone, I have access to my email but am not often logged into anonaddy. The signature in the URL should be sufficient to prove that the email was sent to a recipient who is authorized to disable the alias. In my eyes, this should function the same way that you can unsubscribe from a mailing list with a unique URL without logging in.

I understand this could be seen as a risk, since you can easily forward the anonaddy forwarded emails to other people. In that case, simply send a notification to the user when an alias has been deactivated.

Thank you.

**Comment by ririko5834** [comment]
Yeah,
maybe that option could be in settings if you want to allow performing alias actions using the email header link and turned off by default so only users who want it can enable it

**Comment by ririko5834** [comment]
It would be nice also if there was a link to delete this alias

**Comment by willbrowningme** [comment]
Thanks for the suggestion. I probably won't implement this until I have a reliable way to automatically remove the banner from replies, just to be safe.
